mllib/src/main/scala/org/apache/spark/ml/feature/CountVectorizer.scala

```diff
@@ -127,6 +146,9 @@ class CountVectorizer(override val uid: String)
   /** @group setParam */
   def setMinTF(value: Double): this.type = set(minTF, value)
 
+  /** @group setParam */
+  def setBinary(value: Boolean): this.type = set(binary, value)
+
   setDefault(vocabSize -> (1 << 18), minDF -> 1)
```

Comment:
I have a question about this comment. If I move the setDefault from the trait to CountVectorizer clss, the CounterVectorizerModel should also have the setDefault, right? 

L99, setDefault(minTF -> 1) is in the trait too. Shall we move it out?